jwst/pixel_replace/pixel_replace.py

```diff
@@ -316,6 +317,9 @@ def fit_profile(self, model):
             )[range(*profile_cut)]
 
             replace_mask = np.where(~np.isnan(cleaned_current))[0]
+            if len(replace_mask) == 0:
+                log.info(f"Profile in {self.LOG_SLICE[dispaxis - 1]} {ind} has no valid values - skipping.")
```

Comment:
Should this perhaps be either a debug or warning level message?

Reply:
I considered a warning, but it's not really clear to me if this catch is going to trigger more frequently for cases like reference file edge effects or cases where someone's data has an entire row affected by CRs/other bad effects. I could see it going up to warning.